### ISSUE TITLE
Chrome - Cannot login to GUI

### ISSUE THREAD

**Issue by zaggash** [opnsense/core#3633]
**Describe the bug**

I cannot login to the GUI with chrome
Version 76.0.3809.100 (Build officiel) (64 bits)

To use the GUI I have to use Msft Edge

**To Reproduce**
Go to the login page
Enter the password
Press Login
Wait indefinitly

**Expected behavior**
The dashboard should appear.

**Relevant log files**
I have these logs  from the GUI logs :

Aug 7 01:36:01 | lighttpd[22833]: (mod_openssl.c.1975) SSL: 1 -1 error:1409F07F:SSL routines:ssl3_write_pending:bad write retry
-- | --
Aug 7 01:31:21 | lighttpd[22833]: (mod_openssl.c.1762) SSL: -1 5 54 Connection reset by peer
Aug 7 01:31:07 | lighttpd[22833]: (mod_openssl.c.1762) SSL: -1 5 54 Connection reset by peer
Aug 7 01:31:07 | lighttpd[22833]: (mod_openssl.c.1762) SSL: -1 5 54 Connection reset by peer

**Additional context**
I tried either the IP or the hostname.domainname
I have HTTPS GUI, changed the port to 8443 and disabled the 80-8443 redirection.

**Environment**
OPNsense 19.7.2-amd64
FreeBSD 11.2-RELEASE-p12-HBSD
OpenSSL 1.0.2s 28 May 2019

**Comment by lrosenman** [comment]
works for me on Mac: Version 76.0.3809.100 (Official Build) beta (64-bit)

**Comment by lrosenman** [comment]
Do you have a valid certificate installed?

**Comment by lrosenman** [comment]
and what exact version of OPNSense?

**Comment by zaggash** [comment]
I have the same behaviour on my Mac too, so maybe something is wrong on my env.

The version is :
```
Environment
OPNsense 19.7.2-amd64
FreeBSD 11.2-RELEASE-p12-HBSD
OpenSSL 1.0.2s 28 May 2019

```

Yes I have a SSL certifcate activated, signed by a CA created on OPNsense itself.
I installed it on my desktop machine. All the browsers turn green.

With the previous Chrome version, I had almost the same issue but I could press the "stop" button then "reload page" and the dashboard appeared.

Since the last Chrome update, I cannot log at all.

**Comment by zaggash** [comment]
Sorry about the noise.

I finally solved it.

I found that if I use chrome with my account it doesn't work even on Private mode without any extensions.
So I tried to activate the Guest mode and it works.
I assumed the issue was on my settings.


I played with my Chrome settings and finally I made it works on my account.

Go to Settings -->Advanced --> Privacy and security --> Sync and Google services --> Safe Browsing (protects you and your device from dangerous sites)

If this option is activated, I cannot login, when I deactivate the option, the problem is solved.

Hope it will help someone !

**Comment by mark-visser** [comment]
@zaggash Have you found out why this is happening? We're also experiencing the same issue for some of our accounts (not only in this software). We also found out that turning the safe browsing off helps the accounts that have a problem. Turning it on for accounts that did not have the issue before does not break it.

I think it has something to do with Chrome 76 in combination with sending passwords for certain accounts.

**Comment by zaggash** [comment]
@mark-visser Nop, I just tried all relevant options in chrome and found this one blocking my access.
I do not have the root cause of the issue.

**Comment by mark-visser** [comment]
@zaggash We've debugged it a bit further... Might be worth checking if our findings also apply to you.

We've found out that if your logged in Google Account (So in Chrome, you had safe browsing enabled, so you must have been logged in to Chrome) uses the same password as the one you are submitting, it fails.

If I change either password (so the Google password) or the password for the service, everything works again.

If you have time to check if that also applies to you, that would be great.
